airbyte/_util/meta.py

```diff
@@ -30,6 +30,20 @@ def is_ci() -> bool:
     return "CI" in os.environ
 
 
+@lru_cache
+def is_langchain() -> bool:
+    """Return True if running in a Langchain environment.
+
+    This checks for the presence of the 'langchain-airbyte' package.
+
+    TODO: A more robust check would inspect the call stack or another flag to see if we are actually
+          being invoked via LangChain, vs being installed side-by-side.
+
+    This is cached for performance reasons.
+    """
+    return "langchain-airbyte" in sys.modules
```

Comment:
would this also work if someone is using pip install ?

Reply:
once you check-in, i can try it out.

Reply:
Yes. I've just tested and confirmed. The `sys.modules` list contains all Python packages which have been imported into the Python runtime since launch. This includes `langchain_airbyte` _after_ the user has run `import langchain_airbyte` or `from langchain_airbyte import document_loaders`, etc.